src/utils/daos.c

```diff
@@ -1328,6 +1333,7 @@ do { \
 	"	  delete-acl       delete an entry from a container's ACL\n" \
 	"	  set-owner        change the user and/or group that own a container\n" \
 	"	  stat             get container statistics\n" \
+	"	  check            check objects consistency in container\n" \
```

Comment:
check is very general, and could apply to objects, snapshots, etc.
how about check-obj or verify-obj?

Reply:
Originally, I used "verify-objects", but according to the description in DAOS-6340, my understand is that Johann wants something like DAOS filesystem checker. If that is correct, "check" may be more consistent with others name, such as "e2fsck" or "lfsck".
@johannlombardi , what is your suggestion?

Reply:
The intent is to verify more and more under "container check", so i would stick to this.